plugins/com.google.cloud.tools.eclipse.appengine.newproject/plugin.xml

```diff
@@ -11,9 +11,17 @@
             category="com.google.cloud.tools.eclipse.appengine.wizards">
          <description>%wizard.description</description>
          <selection class="org.eclipse.core.resources.IResource"/>
+         <keywordReference id="com.google.cloud.tools.eclipse.appengine.ui.GCPKeyword"/>
```

Comment:
(I apparently didn't click the submit button, so this wasn't sent yesterday)

> Is there API to check that?

 Nothing beyond using the IExtensionRegistry directly.